### PR TITLE
[Misc. DevOps] Fix data seeding for MariaDB

### PR DESCRIPTION
**Pull request by seanpreston**
# Purpose

This PR seeds data correctly into MariaDB so it can be used in local testing without the test runner.

# Changes

- Adds `mariadb_setup.py` to seed data into `mariadb_example` when being used by `make integration-env`
- Splits `mariadb_example.sql` into two separate files, one for the schema code and the other for the data insertions
- References the schema SQL file within the Docker Compose config, so as to always create the database and schema when bringing up `mariadb_example`
- References the data SQL file within the test fixtures and `mariadb_setup.py`, so as to always seed data into the database in either scenario we want to use it

# Checklist

- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md))
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

### CODE REVIEW

**Review by pattisdr — CHANGES_REQUESTED** [review]
Thanks for your work to seed the database so we can use it in local testing!  Just a few requested changes -

File: tests/integration_tests/setup_scripts/mariadb_setup.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    engine = get_db_engine(database_uri=uri)
+    SessionLocal = get_db_session(
+        engine=engine,
+        autocommit=True,
+        autoflush=True,
+    )
+    session = SessionLocal()
+
+    with open("./data/sql/mariadb_example_data.sql", "r") as query_file:
```
[the remark this answers is not in the document]
Comment (seanpreston) [reply tests/integration_tests/setup_scripts/mariadb_setup.py:52]: Great spot, thanks!

File: tests/fixtures/mariadb_fixtures.py [hunk 2]
```diff
@@ -111,79 +112,14 @@ def truncate_all_tables(db_session):
...
-    [mariadb_integration_session.execute(stmt) for stmt in statements]
+    with open("./data/sql/mariadb_example_data.sql", "r") as query_file:
+        lines = query_file.read().splitlines()
+        filtered = [line for line in lines if not line.startswith("--")]
+        queries = " ".join(filtered).split(";")
+        [
+            mariadb_integration_session.execute(f"{sqlalchemy.text(query.strip())};")
+            for query in queries
+            if query
+        ]
```
[the remark this answers is not in the document]
Comment (pattisdr) [reply tests/fixtures/mariadb_fixtures.py:123]: thanks for reticketing!

**Review by pattisdr — APPROVED** [review]
looks great thanks 🏆

**Pull request merged**